docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -13,5 +13,25 @@ There you can find links to upgrade notes for other versions too.
 
     **Be careful, this upgrade will remove sessions**
 
+## [shopsys/project-base]
```

Comment:
We don't use project-base in upgrade notes. This change should be in framework section.

Reply:
Hello @DavidOstrozlik and @boris-brtan,

I have updated upgrade notes as requested.